src/main/engine/util/currencies.ts

```diff
@@ -27,6 +27,7 @@ export const currencies: Currency[] = [
     { code: 'bob', symbols: ['$b'], name: 'Bolivia Bolíviano', exponent: 2 },
     { code: 'brl', symbols: ['r$'], name: 'Brazil Real', exponent: 2 },
     { code: 'bsd', symbols: ['bs$'], name: 'Bahamas Dollar', exponent: 2 },
+    { code: 'btn', symbols: ['Nu.'], name: 'Bhutanese Ngultrum', exponent: 2 },
```

Comment:
after some testing, I couldn't match price (using `String.parsePrice`) with these `symbols` (but only with code, eg. `btn10`). Checking the source code, I found out that pricing matching (which is basically searching by code/symbol, in this list) is case sensitive.

Even though the [price regexp built](https://github.com/ubio/engine/blob/7d88e02191fc8f045be771dec3d655a9e963c89d/src/main/engine/util/price.ts#L33) ignores it, there's still another [piece of code](https://github.com/ubio/engine/blob/7d88e02191fc8f045be771dec3d655a9e963c89d/src/main/engine/util/price.ts#L70-L76) that matches (and formats) price output, which will use first nonempty string as "symbol" (in this example, could be one of "btn" or "nu.", most likely lowercased).

So, either we ensure these `symbols` array always have lowercased values, or we change the other matching function to ignore case when matching by code/symbol.

Reply:
nice catch, thanks a lot. I wasn't paying attention to such details and just took over the code as I found it on the internet. Probably it would make sense to make it case insensitive as this may happen again when adding new currencies, but I suggest we proceed as is not to block deployment of capsolver.